python-engine-test/src/test/java/io/deephaven/engine/table/impl/select/TestConditionFilter.java

```diff
@@ -1,440 +0,0 @@
-/**
- * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
- */
-package io.deephaven.engine.table.impl.select;
-
-import io.deephaven.base.verify.Require;
-import io.deephaven.configuration.Configuration;
-import io.deephaven.engine.table.impl.lang.QueryLanguageFunctionUtils;
-import io.deephaven.engine.rowset.RowSet;
-import io.deephaven.engine.rowset.RowSetBuilderSequential;
-import io.deephaven.engine.rowset.RowSetFactory;
-import io.deephaven.io.log.LogLevel;
-import io.deephaven.io.logger.StreamLoggerImpl;
-import io.deephaven.util.process.ProcessEnvironment;
-import io.deephaven.compilertools.CompilerTools;
-import io.deephaven.engine.table.Table;
-import io.deephaven.engine.table.lang.QueryScopeParam;
-import io.deephaven.engine.table.lang.QueryScope;
-import io.deephaven.engine.util.PythonDeephavenSession;
-import io.deephaven.engine.util.PythonScopeJpyImpl;
-import io.deephaven.engine.table.ColumnSource;
-import io.deephaven.jpy.PythonTest;
-import org.apache.commons.lang3.exception.ExceptionUtils;
-import org.jpy.PyInputMode;
-import org.jpy.PyModule;
-import org.jpy.PyObject;
-import org.junit.After;
-import org.junit.Assert;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.function.Predicate;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-
-import static io.deephaven.engine.table.impl.select.FormulaTestUtil.*;
-import static org.jpy.PyLib.getMainGlobals;
-import static org.junit.Assert.fail;
-
-@Ignore // TODO (deephaven-core#734)
```

Comment:
https://github.com/deephaven/deephaven-core/issues/734 Says that someone needs to look into why these are failing before deciding to delete.  Was this analysis done?